project-management/naming-things.md

```diff
@@ -0,0 +1,67 @@
+# The subtle art of naming things
+
+> There are only two hard things in Computer Science: cache invalidation and
+> naming things.
+> - [As quoted by Martin Fowler](https://martinfowler.com/bliki/TwoHardThings.html)
+
+We here at DataMade have worked laboriously over the past couple years to get
+our deployment practices setup in such a way that things are generally
+automated. One of the hard lessons was finding all the ways in which naming
+some component in the deployment pipeline could come back to bite us when we
+tried to automate it. This is an attempt at compiling a handy little guide to
+the practices that we've come up with for naming things so that when you're
+getting ready to start deploying the project you're working on, things should
+"just work".
+
+### Github repository
+
+Repositories should typically follow directory naming standards, which is to say
+they should be lowercase and have words delimited with hyphens (`-`).
```

Comment:
Previously this section didn't provide any guidance on what case or punctuation was appropriate in a repo name, so I added some here.